tests/serialization/test_storage.py

```diff
@@ -211,7 +213,6 @@ def test_azure_serialize_with_flows():
     assert serialized["__version__"] == prefect.__version__
     assert serialized["container"] == "container"
     assert serialized["blob_name"] == "name"
-    assert serialized.get("connection_string") is None
```

Comment:
I think you are right to remove this line, but is there a reason why you didn't include a new assertion for the `connection_string_secret`?